### PR TITLE
chore: Markdownlint double trailing space

### PR DESCRIPTION
**Pull request by nschonni**
### Description

From upstream. This rule acts a little like Prettier to clean up double trailing spaces. This are allowed since they can be used to create BR tags, but most cases this is unintentional

### CODE REVIEW

**Review by cw118 — APPROVED** [review]
Looks good for the French pages, thanks so much @nschonni !

File: files/es/learn/html/multimedia_and_embedding/images_in_html/index.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ Pero esto no tiene sentido, solo hace que el navegador trabaje más buscando la
 
 > **Advertencia:** La mayoría de imágenes tienen derechos de autor. **No** muestres una imagen en tu página web a menos que:
 >
-> 1\) seas dueño de la imagen,  
-> 2\) tengas permiso escrito explícito del dueño de la imagen o  
-> 3\) tengas suficientes pruebas de que la imagen es de dominio público.  
+> 1\) seas dueño de la imagen,
+> 2\) tengas permiso escrito explícito del dueño de la imagen o
+> 3\) tengas suficientes pruebas de que la imagen es de dominio público.
```
Comment (Graywolf9) [inline files/es/learn/html/multimedia_and_embedding/images_in_html/index.md:77]:
```suggestion
> 1. seas dueño de la imagen,
> 2. tengas permiso escrito explícito del dueño de la imagen o
> 3. tengas suficientes pruebas de que la imagen es de dominio públic
```

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es`, thank you!

**Review by yin1999 — APPROVED** [review]
LGTM for l10n-zh

**Review by sashasushko — APPROVED** [review]
Approve from RU. Thanks 🙏

**Review by potappo — APPROVED** [review]
LGTM for ja, thank you!

**Review by yujo11 — APPROVED** [review]
LGTM for ko, thank you!

@mdn/yari-content-ko

**Comment by SphinxKnight** [comment]
@mdn/core-dev this is present upstream https://github.com/mdn/content/blob/main/.markdownlint-cli2.jsonc#L165-L171 is there something blocking from merging here?

**Comment by yin1999** [comment]
Hi @bsmth, could you merge this?

As @SphinxKnight points out "this rule is present upstream". We should use this to make the format more rigorous.

**Comment by bsmth** [comment]
> Hi @bsmth, could you merge this?

Merging shortly, thanks all 🙌🏻

**Pull request merged**